src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/model/UserGroup.kt

```diff
@@ -11,5 +13,6 @@ data class UserGroup(
   val groupName: String,
 ) {
   constructor(g: Group) : this(g.groupCode, g.groupName)
+  constructor(g: GroupORModel) : this(g.groupCode, g.groupName)
```

Comment:
I think we should have a better way to identify Group and GroupORModel

Reply:
TODO - rename to GroupWithChildGroups - or similar? also move to same package as others

Reply:
TODO - Identify where using groups(and poss roles) for a user - should we call the new method that creates a jpa style structure?  How do we identify and structure this in the code?

Reply:
Suggestion for:
Model -> domain (ORM like tables) - with associated calls to the db to create the entity structure we need
           -> entity (matches the database tables)